client/src/dummyData/dropDownItems.js

```diff
@@ -0,0 +1,21 @@
+const industries = [
+    { id: 0, name: '' },
+    { id: 1321, name: 'Technology' },
+    { id: 21423, name: 'Marketing' },
+    { id: 42342, name: 'Engineering' },
+    { id: 21342, name: 'Art' },
+    { id: 93082, name: 'Film' }
+]
+
+const locations = [
+    { id: 0, name: '' },
+    { id: 123, name: 'New York' },
+    { id: 431, name: 'California' },
+    { id: 903, name: 'Georgia' },
+    { id: 223, name: 'Florida' }
+]
+
+export {
+    industries,
+    locations
+}
```

Comment:
Honestly dunno about putting dummydata in separate folder, as I tend to put it directly in state before hooking it up, but it doesn't matter that much to block anything